scripts/generateAssetData/generateRelatedAssetIndex.ts

```diff
@@ -2,7 +2,7 @@ import { generateRelatedAssetIndex } from './generateRelatedAssetIndex/generateR
 
 const main = async () => {
   try {
-    await generateRelatedAssetIndex()
+    await generateRelatedAssetIndex(true)
```

Comment:
q: are we changing this to false before merging now that regen has been done? 

https://github.com/shapeshift/web/blob/d2246f0d1d45bbf06ea882ee94f6e6463da61e4d/scripts/generateAssetData/generateRelatedAssetIndex/generateRelatedAssetIndex.ts#L280-L281

Reply:
This is only specifcally run via `yarn generate:related-asset-index` I always change to true because I am running it to actually rebuild. Feels like it should always be `true` for the script unless I am confused on things...

Reply:
This may have borked things unfortunately. 

Don't exactly remember what full rebuild entails, but one thing for sure is it ditches the current ones before regenerating, so there could be something wrong with a regen from scratch we haven't noticed since this is never `true`. Not sure if we still want it at this stage tbh, since we never fully rebuild, we could just remove alongside associated logic?

Reply:
https://github.com/shapeshift/web/pull/9924/commits/43e0de7b78e57eac6d9d29b0fdab2f5505fbdff1